src/plugin.ts

```diff
@@ -1,4 +1,4 @@
-import pluginConfig from "./config";
+import pluginConfig, { VueGtmUseOptions } from "./config";
```

Comment:
Please revert these `VueGtmUseOptions` changes in this file
They don't seem to have anything to do with your fix

Reply:
Your Pick workaround is not required if you do it like this

Reply:
But I can revert this change and add compability "to the Pick"

Reply:
I understand what you mean, but I think I remember that it was a technical design decision

Reply:
All right, sorry for interference, I just found it complicated to work with. Gonna revert it